django/contrib/admin/checks.py

```diff
@@ -771,10 +771,11 @@ def _check_readonly_fields_item(self, obj, field_name, label):
             except FieldDoesNotExist:
                 return [
                     checks.Error(
-                        "The value of '%s' is not a callable, an attribute of "
-                        "'%s', or an attribute of '%s'."
+                        "The value of '%s' refers to '%s', which is not a callable, "
+                        "an attribute of '%s', or an attribute of '%s'."
```

Comment:
All system checks are documented, so any change in the error message should be reflected in the `docs/ref/checks.txt`.

Reply:
Thank you, I did not realize we were copying error messages into the docs. I updated those as well! :musc